src/NServiceBus.Core.Analyzer.Tests/Sagas/RewriteConfigureHowToFindSagaFixerTests.cs

```diff
@@ -0,0 +1,474 @@
+    public class Msg5 : ICommand {}
+}";
+
+            return Assert(original, expected);
+        }
+
+        //        [Test]
+        //        public Task TabsVsSpaces()
+        //        {
+        //            var original =
+        //@"using System;
+        //using System.Threading.Tasks;
+        //using NServiceBus;
+        //namespace SomeNamespace
+        //{
+        //    public class MySaga : Saga<MyData>, IAmStartedByMessages<Msg1>, IAmStartedByMessages<Msg2>
+        //    {
+        //        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<MyData> mapper)
+        //        {
+        //            mapper.ConfigureMapping<Msg1>(msg => msg.CorrId).ToSaga(saga => saga.CorrId);
+        //            mapper.ConfigureMapping<Msg2>(msg => msg.CorrId).ToSaga(saga => saga.CorrId);
+        //        }
+        //        public Task Handle(Msg1 message, IMessageHandlerContext context) => throw new NotImplementedException();
+        //        public Task Handle(Msg2 message, IMessageHandlerContext context) => throw new NotImplementedException();
+        //    }
+        //    public class MyData : ContainSagaData
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //    public class Msg1 : ICommand
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //    public class Msg2 : ICommand
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //}".Replace("    ", "\t");
+
+        //            var expected =
+        //@"using System;
+        //using System.Threading.Tasks;
+        //using NServiceBus;
+        //namespace SomeNamespace
+        //{
+        //    public class MySaga : Saga<MyData>, IAmStartedByMessages<Msg1>, IAmStartedByMessages<Msg2>
+        //    {
+        //        protected override void ConfigureHowToFindSaga(SagaPropertyMapper<MyData> mapper)
+        //        {
+        //            mapper.MapSaga(saga => saga.CorrId)
+        //                .ToMessage<Msg1>(msg => msg.CorrId)
+        //                .ToMessage<Msg2>(msg => msg.CorrId);
+        //        }
+        //        public Task Handle(Msg1 message, IMessageHandlerContext context) => throw new NotImplementedException();
+        //        public Task Handle(Msg2 message, IMessageHandlerContext context) => throw new NotImplementedException();
+        //    }
+        //    public class MyData : ContainSagaData
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //    public class Msg1 : ICommand
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //    public class Msg2 : ICommand
+        //    {
+        //        public string CorrId { get; set; }
+        //    }
+        //}".Replace("    ", "\t");
+
+        //            return Assert(original, expected);
+        //        }
+
```

Comment:
Do we want to leave this test code commented out here?

Reply:
Nope
```suggestion
```